### PR TITLE
image: move idle and nosmt to aws-only images

### PR DESCRIPTION
**Pull request by derpsteb**
### Context

Amends #2291. We don't want these options on other CSPs. This is temporary until AWS fixed some background issues.

### Proposed change(s)

- Move `idle=poll mitigations=auto` options to aws-only config

### Additional info
- Pipeline: https://github.com/edgelesssys/constellation/actions/runs/6070948107

### CODE REVIEW

**Review by malt3 — DISMISSED** [review]

File: image/mkosi.conf.d/mkosi.aws.conf [hunk 1]
```diff
@@ -1,2 +1,5 @@
 [Match]
 PathExists=../.csp/aws
+
+[Output]
+KernelCommandLine=mitigations=auto,nosmt idle=poll
```
Comment (thomasten) [inline image/mkosi.conf.d/mkosi.aws.conf:5]: will this be merged with the KernelCommandLine field in mkosi.conf?

Comment (thomasten) [inline image/mkosi.conf.d/mkosi.aws.conf:5]: remove nosmt, no?

**Review by malt3 — COMMENTED** [review]
As discussed, we will test the actual behavior of mkosi and not take any chances.

**Comment by derpsteb** [comment]
Manual test on AWS succeeded. `cat /proc/cmdline` showed that the expected arguments are set.

Smoketest on Azure: https://github.com/edgelesssys/constellation/actions/runs/6072057303

**Review by malt3 — APPROVED** [review]

**Review by thomasten — APPROVED** [review]

**Pull request merged**